test-suites/pool-flashloan.spec.ts

```diff
@@ -253,6 +255,39 @@ makeSuite('Pool: FlashLoan', (testEnv: TestEnv) => {
       reservesAfter.sub(feesToProtocol).mul(liquidityIndexBefore).div(currentLiquidityIndex)
     ).to.be.closeTo(reservesBefore, 2);
   });
+
+  it('Disable ETH flashloan and takes an ETH flashloan (revert expected)', async () => {
+    const { pool, configurator, helpersContract, weth, deployer } = testEnv;
+
+    expect(await configurator.setReserveFlashLoaning(weth.address, false));
+
+    let wethFlashLoanEnabled = await helpersContract.getFlashLoanEnabled(weth.address);
+    expect(wethFlashLoanEnabled).to.be.equal(false);
+
+    let reserveData = await helpersContract.getReserveData(weth.address);
+
+    const totalLiquidityBefore = reserveData.totalAToken;
+
+    const flashBorrowedAmount = totalLiquidityBefore;
+
+    await expect(
+      pool.flashLoan(
+        _mockFlashLoanReceiver.address,
+        [weth.address],
+        [flashBorrowedAmount],
+        [0],
+        _mockFlashLoanReceiver.address,
+        '0x10',
+        '0'
+      )
+    ).to.be.reverted;
```

Comment:
Lets validate it throws the specific error

Reply:
updated